### ISSUE TITLE
Whether to add hidden text in login script

### ISSUE THREAD

**Issue by qiyue0421** [Eugeny/tabby#7766]
![image](https://user-images.githubusercontent.com/33716216/211232472-f4c1bd58-163c-4b3e-84c1-5724d3140550.png)
If some sensitive information is in the text, can you actively hide it? I look forward to your feedback. Thank you

**Comment by gitthangbaby** [comment]
Good idea, but also i'd like to parse the existing password via variable, that would be safe and invisible.

**Comment by shanelord01** [comment]
Yes - please give me the ability to use a saved password via a variable

**Comment by szplovezxm** [comment]
请问你这里登录脚本生效了吗？我这里不管怎么配都不生效
